plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotConnectorSmokeTest.java

```diff
@@ -1482,23 +1482,23 @@ public void testNullBehavior()
         assertThat(query("""
                 SELECT COUNT(*)
                 FROM alltypes
-                WHERE string_col IS NULL"""))
-                .matches("VALUES (BIGINT '0')")
-                .isNotFullyPushedDown(FilterNode.class);
+                WHERE string_col IS NULL OR string_col = 'null'"""))
```

Comment:
Since null handling is enabled for alltypes should this be `AND string_col = 'null'` to verify null handling? i.e. this would have the same result as if the table had null handling disabled.